md/monitoring.md

```diff
@@ -8,7 +8,3 @@ The view is filtered by process and optionally its version.
 It is composed by a pie chart that shows the ratio between cases late, predicted late and on time.
 It also has a list of cases that display the success rate percentage and the time remaining before the target duration.
```

Comment:
It also contains... that displays the percentile of chances for the case to finish within the target defined, as well as the time remaining before the target duration.